db/migrations/20210427123109_migrate_subscription.js

```diff
@@ -0,0 +1,193 @@
+require('dotenv').config()
+
+const Stripe = require('stripe')
+
+const { v4 } = require('uuid')
+
+const t_coupon = 'circle_coupon'
+
+const t_invitation = 'circle_invitation'
+
+const t_price = 'circle_price'
+
+const t_subscription = 'circle_subscription'
+
+const t_subscription_item = 'circle_subscription_item'
+
+const t_user = 'user'
+
+exports.up = async (knex) => {
+  const secret = process.env['MATTERS_STRIPE_SECRET']
+
+  if (!secret) {
+    console.error('Stripe secret is not provided')
+    return
+  }
+
+  const stripeAPI = new Stripe(secret, { apiVersion: '2020-03-02' })
+
+  const findFirst = async ({ table, where }) => {
+    return knex.select().from(table).where(where).first()
+  }
+  const insert = async ({ table, data }) => {
+    const [record] = await knex(table).insert(data).returning('*')
+    return record
+  }
+  const update = async ({ table, where, data }) => {
+    const [record] = await knex(table).where(where).update(data).returning('*')
+    return record
+  }
+
+  // fetch accepted invitations
+  const invts = await knex
+    .select()
+    .from(t_invitation)
+    .where({ accepted: true })
```

Comment:
skip migrated invitations (`subscription_item_id` ) if we retry?

Reply:
Added.